lib/transport/selenium-webdriver/service-builders/chrome.js

```diff
@@ -38,7 +38,11 @@ class ChromeServiceBuilder extends BaseService {
       this.service.addArguments(...this.cliArgs);
     }
 
-    this.service.enableChromeLogging().enableVerboseLogging();
+    if (process.platform === 'win32') {
+      this.service.enableVerboseLogging();
+    } else {
+      this.service.enableChromeLogging().enableVerboseLogging();
+    }
```

Comment:
Maybe write it like this?

```js
 this.service.enableVerboseLogging();

if (process.platform !== 'win32') {
  this.service.enableChromeLogging()
} 
```

Reply:
fixed in [`cbcd1dc`](https://github.com/nightwatchjs/nightwatch/pull/3125/commits/cbcd1dc3e7e1409b768731558b14d26e7ec0253a)